webapp/src/ts/modules/error/error.component.ts

```diff
@@ -9,18 +10,24 @@ const errors = {
   404: {
     title: 'error.404.title',
     description: 'error.404.description'
-  }
+  },
+  418: {
+    title: 'error.418.description',
+    description: 'error.418.description'
+  },
 };
 
 @Component({
   templateUrl: './error.component.html'
 })
 export class ErrorComponent implements OnInit {
   error;
+  translationsLoaded;
 
-  constructor(private route: ActivatedRoute) {}
+  constructor(private route: ActivatedRoute, private translateService:TranslateService,) {}
 
   ngOnInit() {
     this.error = errors[this.route.snapshot.params.code] || errors['404'];
+    this.translationsLoaded = this.translateService.instant('error.403.description') !== 'error.403.description';
```

Comment:
Does this return true if the translation isn't found, eg: a new language?

Reply:
The translation can eventually load on a slow connection. What we usually do is check the `getTranslationsLoaded` from the store. This value is set when the app.component.ts load the translations. 

```
const subscription = this.store
      .select(Selectors.getTranslationsLoaded)
      .subscribe(isLoaded => this.translationsLoaded = !!isLoaded);
this.subscriptions.add(subscription);

// unsubscribe subscriptions in the destroy function
```

If the connection is slow, there are a few options:
- show the generic message until the translations are loaded, if ever. 
- show a spinner and timeout after a while, then show the generic message. 

IMO The second option creates less confusion for the user.